app/src/main/res/values/themes.xml

```diff
@@ -4,6 +4,6 @@
     <style name="Theme.Fitness_app" parent="Theme.MaterialComponents.Light.NoActionBar">
         <item name="android:statusBarColor">@color/background</item>
         <item name="android:windowBackground">@color/background</item>
+        <item name="android:windowSplashScreenAnimatedIcon">@drawable/ic_launcher</item>
```

Comment:
in this case wouldn't you have two splash screens, on from you composable and one from the system splash  screen

Reply:
Indeed , i wanted to try with the splash screen so i added it.  Will remove it because it looks ugly :0

Reply:
I thought it was agreed that this PR needs a migration to the splash screen api. Is this no longer the case?